x/ibc-hooks/wasm_hook.go

```diff
@@ -24,14 +28,22 @@ type ContractAck struct {
 
 type WasmHooks struct {
 	ContractKeeper *wasmkeeper.PermissionedKeeper
+	hooksKeeper    *keeper.Keeper
 }
 
-func NewWasmHooks(contractKeeper *wasmkeeper.PermissionedKeeper) WasmHooks {
-	return WasmHooks{ContractKeeper: contractKeeper}
+func NewWasmHooks(hooksKeeper *keeper.Keeper, contractKeeper *wasmkeeper.PermissionedKeeper) WasmHooks {
+	return WasmHooks{
+		ContractKeeper: contractKeeper,
+		hooksKeeper:    hooksKeeper,
```

Comment:
nit: maybe call this ibchooksKeeper to avoid confusion in the future

Reply:
renamed